submissions/ses3332002/friends-app/app.js

```diff
@@ -39,7 +40,9 @@ function render() {
     let card = document.createElement("div");
     function sexString(age) {
       const lastNumberOfAge = age.toString().slice(-1);
-      return (lastNumberOfAge == "1") ? "год" : (lastNumberOfAge == "2")||(lastNumberOfAge == "3")||(lastNumberOfAge == "4") ? "года" : "лет";
+      const numberForGod = "1";
+      const numbersForGoda = ["2", "3", "4"];
```

Comment:
whatIsThisNeimingTakoy?)

Reply:
:) I can use more correct and understandable names for this variables, but this names will be long as a train, like numberForYearOptionGod or numberForSuffixlessRussianYearWord. Is it Ok?

Reply:
the variable should reflect the essence of its content)

Reply:
I renamed this vars.